editor/components/skip-to-selected-block/style.scss

```diff
@@ -0,0 +1,27 @@
+.skip-to-selected-block {
```

Comment:
This class name does not adhere to the Gutenberg naming conventions:

https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#naming

It should be corrected to `.editor-skip-to-selected-block`

Reply:
Yep, I've used `editor-` initially then removed waiting for a feedback about the placement of the whole thing,